python/cmsis_svd/parser.py

```diff
@@ -83,12 +84,25 @@ def _parse_field(self, field_node):
         enumerated_values = []
         for enumerated_value_node in field_node.findall("./enumeratedValues/enumeratedValue"):
             enumerated_values.append(self._parse_enumerated_value(enumerated_value_node))
+			
+        bitRange=_get_text(field_node, 'bitRange')
+        bitOffset=_get_int(field_node, 'bitOffset')
+        bitWidth=_get_int(field_node, 'bitWidth')
+        msb=_get_int(field_node, 'msb')
+        lsb=_get_int(field_node, 'lsb')
+        if bitRange is not None:
+            m = re.search('\[([0-9]+):([0-9]+)\]', bitRange)
+            bitOffset = int(m.group(2))
+            bitWidth = 1 + (int(m.group(1)) - int(m.group(2)))     
+        elif msb is not None:
+            bitOffset = lsb;
+            bitWidth = 1 + (msb - lsb);
```

Comment:
Change looks good in content.  A couple style things to comply with PEP8 (https://www.python.org/dev/peps/pep-0008/):

* Variables should be named snake-case (`bitRange` -> `bit_range`)
* Whitespace around operators (`msb = _get_int(...)`).  Assignment for kwargs (e.g. SVDField constructor) is an exception.
* Python doesn't need the semicolons; they should be removed


You can get a quick check here: http://pep8online.com/.  There are some violations (line length, etc.) in the existing code but it can give you a quick linter.  There is a pep8 command-line checker as well.